actionpack/test/controller/render_test.rb

```diff
@@ -276,7 +281,7 @@ def test_dynamic_render_with_file
   def test_dynamic_render
     assert File.exist?(File.join(File.dirname(__FILE__), '../../test/abstract_unit.rb'))
     assert_raises ActionView::MissingTemplate do
-      get :dynamic_render, { id: '../\\../test/abstract_unit.rb' }
+      get :dynamic_render, params: { id: '../\\../test/abstract_unit.rb' }
```

Comment:
Looks like Action Pack tests are passing on Travis, so I'm going to assume that `params` does exist...?